src/FuncHW/ListExtensions.cs

```diff
@@ -12,5 +12,17 @@ public static class ListExtensions
         //add extension for List CountElements: receives Func, returns the number of elements that match the condition
 
         //add extension for List SelectWhereNot: receives Func, returns the list of elements that don't match the condition
+
+        public static T GetLast<T>(this List<T> values, Func<T, bool> func) where T : new()
```

Comment:
what for 'where T : new()'?

Reply:
I may be wrong, but this necessary to return new(); :25